top_films.py

```diff
@@ -1,53 +1,133 @@
 from datetime import datetime
 
 from pyspark.sql import functions as f
+from pyspark.sql.window import Window
 
-from connections import read_tsv
 
-titles_info = read_tsv('title.basics.tsv')
+def join_dataframes(left_df, right_df, condition):
+    """
+    Create dataframe by joining two datasets and condition for joining
+    """
+    return left_df.join(right_df, on=condition, how='inner')
+
 
-rating_info = read_tsv('title.ratings.tsv')
+def prepare_rating_info(left_df, right_df):
+    """
+    Create dataframe about film`s title and rating
+    """
+    df = join_dataframes(left_df, right_df, 'tconst') \
+        .where((f.col('titleType') == 'movie') & (f.col('numVotes') >= 100000)) \
+        .orderBy(f.col('averageRating').desc(), f.col('numVotes').desc())
```

Comment:
we don't need sorting by `f.col('numVotes').desc()`

Reply:
it shows a more accurate rating of films